src/main/java/net/imagej/table/DefaultColumn.java

```diff
@@ -41,12 +41,16 @@
  */
 public class DefaultColumn<T> extends SizableArrayList<T> implements Column<T> {
 
+	/** The type of this column. */
+	private Class<T> type;
+
 	/** The column header. */
 	private String header;
 
 	public DefaultColumn() {}
 
-	public DefaultColumn(final String header) {
+	public DefaultColumn(final Class<T> type, final String header) {
```

Comment:
Note that changing public API breaks backwards compatibility. Of course, `imagej-common` is still in `0.x`, so we have the luxury of doing this, but I would still suggest, in general, to try to find a way to avoid breaking backwards compatibility when possible.